Iggy_SDK/Contracts/Http/Auth/AuthResponse.cs

```diff
@@ -3,5 +3,5 @@ namespace Iggy_SDK.Contracts.Http;
 public sealed class AuthResponse
 {
     public required int UserId { get; init; }
-    public Tokens? Tokens { get; init; } 
+    public required TokenInfo? AccessToken { get; init; }
```

Comment:
No point in making required a property that is nullable aswell.

Reply:
The fix is done.

I ended up changing the _sealed class_ to **record**, since the usage of this data structure is only to be a data container, though.